For consistency's sake

tondro1 opened the pull request:
all event form tests are done, fixed minor bugs in view, forms, and made migrations once to make the tests run

Commit:
all event form tests are done, fixed minor bugs in view, forms, and made migrations once to make the tests run

ImaneChafi reviewed (approved):
Looks good to me!

ImaneChafi reviewed (changes requested):
I see a UnboundLocalError at /dashboard/events/create/ that wasn't there on master, I'll try to first fix the models for creating event on master and then you'll be able to merge!

ImaneChafi committed:
Fixed Events error, I'll add date afterwards

We needed to delete the date field and then create it again because the migration wasn't done on the event

ImaneChafi committed:
EVERYTHING WORKS!!

ImaneChafi committed:
linked creating event button to events

ImaneChafi committed:
Integrated Create Event HTML from profile page

ImaneChafi committed:
Linked back to profile page from the side bar

ImaneChafi committed:
Finished adding create event

Everything works and is linked now for create event, I'll add delete event and edit event before testing it!

ImaneChafi committed:
Merge branch 'event_create_delete' into testing_clean
